### PR TITLE
Update mapping for aria-errormessage

### LINKED ISSUE #138: Update Core-AAM aria-errormessage to take multiple element references
Align Core-AAM with [ARIA PR 1802](https://github.com/w3c/aria/pull/1802/) [issue 1730](https://github.com/w3c/aria/issues/1730)
change aria-errormessage to take multiple element references

IA2 and ATK/AT-SPI already point to multiple nodes
update UIA to have ControllerFor point to an array of references to accessibility nodes

@cookiecrook need text for AX API. Assign back to me with text
Old text is 
AX API | Property: AXValidationError: textual content of the referenced element

### PR DESCRIPTION
**Pull request by cookiecrook**
Closes #138

Update macOS mapping for aria-errormessage to AXErrorMessageElements

# Implementation

* WPT tests: https://github.com/web-platform-tests/wpt/pull/40572
* Implementations (link to issue or when done, link to commit):
   * WebKit: shipping
   * Gecko: ?
   * Blink: [issue](https://bugs.chromium.org/p/chromium/issues/detail?id=1403266)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/core-aam/pull/179.html" title="Last updated on Jun 16, 2023, 3:09 PM UTC (193af70)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/core-aam/179/ed79fdf...193af70.html" title="Last updated on Jun 16, 2023, 3:09 PM UTC (193af70)">Diff</a>

### CODE REVIEW

**Comment by spectranaut** [comment]
I don't see this in the blink code, @cookiecrook -- I don't see an issue for it either. Why did you write "shipping" above?

Looks to me like we need issues for both blink and gecko. I'll submit a test fix for the core-aam tests, however.

**Comment by spectranaut** [comment]
about the tests, I'm keeping the core-aam tests update for now -- so if there is a change to the mapping tables there is a change to the wpt/core-aam tests. We have some hack-y ways to run them and eventually we will translate them to a newer test format.

**Comment by cookiecrook** [comment]
> I don't see this in the blink code, @cookiecrook -- I don't see an issue for it either. Why did you write "shipping" above?

I must’ve read too much into [this comment](https://github.com/w3c/core-aam/issues/138#issuecomment-1546527405).

**Comment by spectranaut** [comment]
ok I found the issue, and I guess this is implementation commitment since @aleventhal commented on it but I'll wait until he sees this to merge: https://bugs.chromium.org/p/chromium/issues/detail?id=1403266&sort=pri&q=aria-errormessage&can=2

**Review by spectranaut — APPROVED** [review]

**Pull request merged**
